scss/_patterns_switch.scss

```diff
@@ -50,6 +50,7 @@ $knob-size: $sp-unit * 3;
       width: $knob-size;
     }
 
+    // stylelint-disable-next-line selector-max-type
```

Comment:
In places like this we should add also some `// FIXME` comment to note that this is not an expected place for element selector and should be fixed.